wpeverest/everest-forms
# Enhancement - Required Indicators for Field Types

### All Submissions:




Previously there was only one indicator for the required field as an asterisk ( * ). This PR helps the users to choose options for the required field as Text Message (Required), Asterisk ( * ), and Custom Text ( User can type any value they need as text ). This will reflect for the users to show in the form fields.

### How to test the changes in this Pull Request:

1. Open **Form Builder** area in Everest Forms.
2. Click on Form Settings.
3. Click on General Tab on Form Settings.
4. Scroll down to Required Field Indicators Fieldset.
5. Choose the required value from the select box. If you choose the **Custom Text** option, then the input box will be shown and you can type any text you want to show as required indicator text.
6. That's all. You can save changes and see the updates in the preview field or form builder label names.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Required Indicators for Field Types
